pandas/io/pickle.py

```diff
@@ -10,15 +10,17 @@
 
 def to_pickle(obj, path, compression='infer', protocol=pkl.HIGHEST_PROTOCOL):
     """
-    Pickle (serialize) object to input file path
+    Pickle (serialize) object to input file path.
```

Comment:
I find it a bit strange to say we pickle the object the a "file path", it's actually to a file (located at the file path), so maybe simplify to "Pickle (serialize) object to file." ?

Reply:
amended 👍